### PR TITLE
Feature/command line tests

### PR DESCRIPTION
**Pull request by vjt**
Based on #59 by @roschaefer.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -4,10 +4,12 @@ require "bundler/gem_tasks"
 # RSpec
 require 'rspec/core/rake_task'
 RSpec::Core::RakeTask.new
-task :default => ['dummy:create', :spec]
+task :default => ['testapp:create', :spec]
```
Comment (roschaefer) [inline Rakefile]: Good! :+1: 

How about a consistent name like `testapp`? We could move it into a folder `testapp` and the database would be named `chronomodel_testapp`?

File: spec/aruba/migrations_spec.rb [hunk 2]
```diff
@@ -2,21 +2,17 @@
 
 describe 'database migrations' do
   context 'after a migration was generated' do
+    before { write_file('config/database.yml',
+      File.read(File.expand_path('fixtures/database_without_username_and_password.yml', __dir__))) }
+
     before { run_simple('bundle exec rails g migration CreateModels name:string') }
 
-    describe 'bundle exec rake db:migrate', type: :aruba do
+    describe 'bundle exec rake db:migrate', :announce_stdout, :announce_stderr, type: :aruba do
```
Comment (roschaefer) [inline spec/aruba/migrations_spec.rb]: Nice! I didn't know about those two :smile:

File: spec/aruba/fixtures/database_with_default_username_and_password.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  encoding: unicode
+  pool: 20
+  database: chronomodel_railsapp
+
+development:
+  <<: *default
+
+test:
+  <<: *default
+
```
Comment (roschaefer) [inline spec/aruba/fixtures/database_with_default_username_and_password.yml]: `git diff 40e0951..HEAD --stat` tells me this file was moved from `overwrite/config/database.yml`. So, no default database.yml anymore?

**Comment by roschaefer** [comment]
Ahh, one more thing maybe: Do you think it's an issue that we expect potentially three database users with root privileges now?

(username: `nil` (resolves to `current_user`?), username: `current_user`, username: `chronomodel`)

File: spec/aruba/fixtures/database_with_default_username_and_password.yml (hunk 3 above)
Comment (vjt) [reply spec/aruba/fixtures/database_with_default_username_and_password.yml]: I have mixed feelings about this. Maybe we should use sane defaults, that is do not provide username & password and use the defaults, and provide a way for the end user to specify username and password on the command line... WDYT?

File: Rakefile (hunk 1 above)
Comment (vjt) [reply Rakefile]: Agreed.

**Comment by vjt** [comment]
We are expecting to find the `chronomodel` and `chronomodel_testapp` database, I do not see a major issue with that. Yet we could use just a single database, but it's minor. On credentials see comment above.

Thanks!

File: spec/aruba/fixtures/database_with_default_username_and_password.yml (hunk 3 above)
Comment (roschaefer) [reply spec/aruba/fixtures/database_with_default_username_and_password.yml]: Sounds reasonable to me. :+1:

**Comment by vjt** [comment]
Merged and released as v0.12.0. Enjoy! :-)

**Pull request merged**